docs/api/web-contents.md

```diff
@@ -1416,6 +1420,46 @@ Executes the editing command `selectAll` in web page.
 
 Executes the editing command `unselect` in web page.
 
+#### `contents.scrollToTop()`
+
+Scrolls to the top of the current `webContents`.
+
+#### `contents.scrollToBottom()`
+
+Scrolls to the bottom of the current `webContents`.
+
+#### `contents.adjustSelection(options)`
+
+* `options` Object
+  * `start` Number - Index at which to begin the adjustment.
+  * `end` Number - Index at which to end the adjustment.
```

Comment:
question: isn't this the shift amount, rather than the start/end indexes?

Reply:
yes and frankly i have no idea why I wrote this so terribly lol